src/scope.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @ignore
+ */
+const dedupe = arr => arr.filter((x, i) => arr.indexOf(x) === i);
```

Comment:
(optional) now we have `Set`, you can do:

`const dedupe = arr => [...new Set(arr)]`

Reply:
I had to change it to `[...(new Set(arr))]`, but yes - using `Set` here makes sense now that we have the polyfill available.

However, doing this requires us to use the `--downlevelIteration` compiler flag which [adds a bit of bloat to the code](https://mariusschulz.com/blog/downlevel-iteration-for-es3-es5-in-typescript#the-downleveliteration-flag). The code can be changed to `Array.from(new Set(arr))` which does not require this flag. Possibly a little more readable too.